content/en/docs/v3.6/tutorials/how-to-watch-keys.md

```diff
@@ -4,9 +4,31 @@ description: Guide to watching etcd keys
 weight: 600
 ---
 
+## Prerequisites
 
+* Install [`etcd` and `etcdctl`](https://etcd.io/docs/v3.6/install/)
+
+## Watching keys
 `watch` to get notified of future changes:
+```
+etcdctl watch $KEY [$END_KEY]
+```
+
+### Options
+```
+-i, --interactive[=false]: interactive mode
+--prefix[=false]: watch on a prefix if prefix is set
+--rev=0: Revision to start watching
+--prev-kv[=false]: get the previous key-value pair before the event happens
+--progress-notify[=false]: get periodic watch progress notification from server
+```
+
+### Options inherited from parent commands
+```
```

Comment:
nit: markdownlint wants fenced code blocks to have a language specified and be surrounded by blank lines

```suggestion

```bash
```